packages/did-core-test-server/suites/did-spec/did-core-properties.js

```diff
@@ -0,0 +1,31 @@
+const utils = require('./utils');
+const {isValidDID} = utils;
+
+const jsonMediaTypes = ['application/did+ld+json', 'application/did+json'];
+
+const generateDidCorePropertiesTests = ({did, resolutionResult}) => {
+  const {didDocument} = resolutionResult;
+
+  it('5.1.1 DID Subject - The value of id MUST be a string that ' +
+    'conforms to the rules in § 3.1 DID Syntax and MUST exist in the root ' +
+    'map of the data model for the DID document.', async () => {
+      expect(didDocument).toHaveProperty('id');
+      expect(isValidDID(didDocument.id)).toBe(true);
```

Comment:
Jest's expect syntax leaves something to be desired... not a fan.

Reply:
I'm a fan! https://jestjs.io/docs/expect see also https://github.com/jest-community/jest-extended

The line you hilighted is asserting the return value of a helper function... you could rewrite the helper function to `assertValidDID` and have it throw instead of return a boolean.

Reply:
> you could rewrite the helper function to assertValidDID and have it throw instead of return a boolean.

Yes, as @shigeya raised in an issue, that should probably be what we do to make the tests easier to read. Granted, I'm not looking forward to going back and rewriting lots of tests. :)